.eslintrc.json

```diff
@@ -1,61 +1,60 @@
 {
   "root": true,
-  "plugins": ["prettier"],
-  "rules": {
-    "prettier/prettier": "warn"
+  "parser": "@typescript-eslint/parser",
+  "parserOptions": {
+    "project": ["tsconfig.*?.json"],
+    "createDefaultProgram": true
   },
-  "extends": ["eslint:recommended", "plugin:prettier/recommended"],
+  "plugins": ["@typescript-eslint"],
+  "extends": [
+    "plugin:@angular-eslint/recommended",
+    "plugin:@typescript-eslint/recommended",
```

Comment:
This includes rules from `eslint:recommended` (see [note](https://typescript-eslint.io/docs/linting/configs/#recommended)).